docs/src/references/c8y-log-management.md

```diff
@@ -114,28 +114,28 @@ OPTIONS:
     -V, --version
             Print version information
 
-On start, `c8y-log-plugin` notifies the cloud tenant of the log types listed in the `CONFIG_FILE`,
+On start, `tedge-log-plugin` notifies the cloud tenant of the log types listed in the `CONFIG_FILE`,
 sending this list with a `118` on `c8y/s/us`.
-`c8y-log-plugin` subscribes then to `c8y/s/ds` listening for logfile operation requests (`522`)
+`tedge-log-plugin` subscribes then to `c8y/s/ds` listening for logfile operation requests (`522`)
 notifying the Cumulocity tenant of their progress (messages `501`, `502` and `503`).
```

Comment:
Another `c8y-log-plugin` contract which isn't valid anymore.

Reply:
Yes someone will have to update this document.

Reply:
Done with https://github.com/thin-edge/thin-edge.io/pull/2213/commits/09f8e40fc990560635484e97809a35477465459e.